src/__tests__/components/CreateTemplate/SuccessNotice.test.tsx

```diff
@@ -0,0 +1,28 @@
+import { screen, waitFor } from "@testing-library/react";
+import SuccessNotice from "@/components/CreateTemplate/SuccessNotice";
+import renderWithMockedProvider from "../../utils";
+import userEvent from "@testing-library/user-event";
+import React from "react";
+
+describe("SuccessNotice", () => {
+  it("should render successNotice page when user punish template successfully", () => {
+    renderWithMockedProvider(<SuccessNotice isOpen={true} onClose={() => void {}} />);
```

Comment:
The only different between renderWithMockedProvider and render(the one from @testing-library/react) is we wrapped redux in renderWithMockedProvider but no redux in the render from the testing library.

So in this case, the render from the testing library should do all these works.

Reply:
> The only different between renderWithMockedProvider and render(the one from @testing-library/react) is we wrapped redux in renderWithMockedProvider but no redux in the render from the testing library.
> 
> So in this case, the render from the testing library should do all these works.

fixed